db/seeds/development.rb

```diff
@@ -0,0 +1,5 @@
+require 'csv'
+
+CSV.foreach("db/seeds/Food_Pantries.csv", { headers: true, header_converters: :symbol, converters: :all}) do |row|
```

Comment:
Line is too long. [114/80]<br>Redundant curly braces around a hash parameter.<br>Space inside } missing.